src/components/MainContent.js

```diff
@@ -62,45 +70,188 @@ export default function MainContent() {
     );
   });
 
+  const tableData = [
```

Comment:
you can create a separate folder, maybe a data folder. put this in there. export it and just use it here.